main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -542,6 +543,8 @@ static public long saveStreamToFile(InputStream stream, File file, SavingUpdate
                 }
             }
             return length;
+        } catch (ZipException aException) {
+            throw new IOException("PPMd compression not supported!");
```

Comment:
It is not clear to me that all `ZipException` instances that could be thrown by this code relate to PPMd compression. Should we not add more specific checks, or forward the error message from the `ZipException`?

Reply:
I will look into this, thanks.